alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -206,7 +206,7 @@ pub fn new<C>(config: &Config<C>, size: &SizeInfo, _window_id: Option<usize>) ->
     let cwd = config
         .working_directory
         .as_ref()
-        .map(|pb| win32_string(&pb.as_path().canonicalize().unwrap()));
+        .map(win32_string);
```

Comment:
Do we still need the `.as_ref()`?

Reply:
Sadly yes - `Option::map` takes `self`, so we need to use `as_ref()` first to get a new `Option<&PathBuf>` to consume.

Reply:
Oh, right, we access something on our config and can't just consume the original value. Nevermind then, thought for a second that we had full ownership over the path.